spec/classes/foreman_proxy_content_spec.rb

```diff
@@ -60,8 +60,10 @@ class { 'foreman_proxy::plugin::pulp':
         it { is_expected.to contain_class('pulpcore').with(manage_apache: false) }
 
         it do
-          is_expected.to contain_foreman__config__apache__fragment('pulpcore')
+          is_expected.to contain_foreman__config__apache__fragment('pulpcore-https')
             .with_ssl_content(%r{ProxyPass /pulp/api/v3 http://127\.0\.0\.1:24817/pulp/api/v3})
+          is_expected.to contain_foreman__config__apache__fragment('pulpcore-http')
+            .with_content(%r{ProxyPass /pulp/content http://127\.0\.0\.1:24816/pulp/content})
```

Comment:
Can you add this `with_content` check two lines above as well? This way the tests ensure that the content app proxy is in http and https fragments.

Reply:
@ehelms Done. I also added a test for the `pulpcore-isos` fragment and renamed its template to be consistent with the naming for content and api templates. Thanks for hte suggestion.